### ISSUE TITLE
Performance on PERMED-01 dataset

### ISSUE THREAD

**Issue by mdelcorvo** [google/deepsomatic#49]
Dear DeepSomatic team,

In a recent benchmarking [study](https://academic.oup.com/bib/article/26/1/bbae697/7960049) the authors evaluated the performance of 20 individual somatic variant callers (including DeepSomatic) across four reference datasets.

It appears that for both SNVs and Indels, DeepSomatic performed exceptionally well on three of these datasets, matching or even surpassing other robust callers such as DRAGEN and NeuSomatic. However, on one dataset ([PERMED-01](https://academic.oup.com/view-large/figure/500645095/bbae697f1.tif)), its performance was notably lower

As stated in the paper:
> However, DeepSomatic and VarNet, two other deep-learning methods, showed poor performance within the PERMED-01 dataset in the detection of somatic SNVs. It was particularly pronounced for DeepSomatic, which demonstrated excellent results in the other three datasets. After investigation, we noticed that this was due to a high number of false-positive somatic variants, which were actually germline variants.

Do you have plans to include this dataset— which appears particularly challenging for detecting somatic variants— in future model updates to deal with this high number of false-positive? If it weren’t for this, DeepSomatic could easily rank among the top somatic variant callers overall.

Thanks

**Comment by AndrewCarroll** [comment]
Hi @mdelcorvo 

Thank you for the question. To give a bit more context to Kishwar's answer - we had noticed the results in that study. The first step we want to take is to understand why performance is lower on that sample. Specifically, we want to look at why the benchmarks diverge, at how the truth on that sample is determined, and then draw a conclusion about whether the benchmark is sufficiently reliable for evaluation or for training.

After that, we want to understand why DeepSomatic performs differently - is it aspects of pre-processing or something learned by the neural network. If so, what learned concept (or not learned concept) is responsible for the divergence. At the end of that process, we'll understand if this is something that needs to be addressed (a real effect on accuracy) or it represents something about the benchmark that is artifactual or doesn't generalize.

Finally, if we conclude this is a real effect on accuracy, we will address this through training, either by using that dataset, or by identifying a similar dataset, training on that, and using PERMED-01 as a holdout.

This process is generally how we go about the process of improving our models (DeepVariant, DeepConsensus, or DeepSomatic) as new benchmark datasets are available.
